src/renderer/screens/exchange/Swap2/Form/FormSelectors/FormLabel.js

```diff
@@ -4,7 +4,7 @@ import Text from "~/renderer/components/Text";
 
 export function FormLabel({ children }: { children?: React$Node }) {
   return (
-    <Text ff="Inter|SemiBold" color="palette.text.shade140" uppercase fontWeight="600">
+    <Text ff="Inter|SemiBold" color="palette.text.shade140" fontWeight="600">
```

Comment:
shade goes up to 100, I'm not sure what this is but either 100 or 40 I guess?

Reply:
Nice catch, didn't see this one as I just removed the uppercase props. I took a look at the Figma and catch that 50 was requested by the design team. I'm going to change this one.